src/bufr_interface.F90

```diff
@@ -53,7 +58,7 @@ subroutine cwbmg_c( bmg, nmb, iret ) bind(C, name='cwbmg')
     end subroutine cwbmg_c
 
     !> @fn bufr_interface::ccbfl_c::ccbfl_c()
-    !> Wraps BUFRLIB ccbfl() function.
+    !> Wraps NCEPLIBS-bufr ccbfl() function.
```

Comment:
Two points:
1. We don't need to say "NCEPLIBS-bufr ccbfl() function", we can just say "Wraps ccbfl()."
2. This is not a good brief because the user will not know what the ccbfl() function does. Instead, these functions should have briefs like the functions they wrap, and then the fact that they are wrapping should be mentioned but not in the brief. Something like:

    !> @fn bufr_interface::ccbfl_c::ccbfl_c()
    !> Closes all system files that were opened via previous calls to subroutine cobfl().
    !>
    !> Wraps ccbfl().

Reply:
But when the user hovers above the function name, he or she sees "Wraps gobbledy-goop" which is not particularly helpful.

The brief needs to tell what the function does in a way the user quickly understands, so he can scan the functions listed in the docs as say "ah HA! THAT'S the one I want!"

Reply:
OK, got it.  I've already done this in bufr_interface.F90 and bufrlib.F90, BTW.  But I'll go ahead and add those now in bufr_c2f_interface.F90 as well, which never had them.